distribution/ddf/src/main/descriptors/bin.xml

```diff
@@ -13,16 +13,24 @@
 -->
 <assembly>
 
-  <id>bin</id>
+    <id>bin</id>
 
-  <formats>
-    <format>zip</format>
-  </formats>
+    <formats>
+        <format>zip</format>
+    </formats>
 
- 
     <componentDescriptors>
         <componentDescriptor>target/dependencies/common-bin.xml</componentDescriptor>
     </componentDescriptors>
-    
-  
+
+    <dependencySets>
+        <dependencySet>
+            <outputDirectory>system/ddf/platform/osgi/platform-osgi-configadmin/${project.version}
+            </outputDirectory>
+            <includes>
+                <include>ddf.platform.osgi:platform-osgi-configadmin</include>
+            </includes>
+        </dependencySet>
+    </dependencySets>
```

Comment:
You might advise me to create a feature so we don't need this snippet. Then Karaf's feature service could generate the `system/` directory with all relevant artifacts, ours included. I would argue this is not a feature but a system-level bundle. Installing it in this fashion ensures it cannot be tampered with, and IMHO, is simpler. There is a deeper problem present here that needs to be addressed in the future should we wish to continue customizing boot bundles.